README.md

```diff
@@ -1,5 +1,10 @@
+
 # [![OWASP Logo](./images/owasp_logo.png) OWASP Amass](https://owasp.org/www-project-amass/)
 
+<p align="center">
+  <img src="https://github.com/jhaddix/Amass-1/blob/master/doc/amass.gif">
```

Comment:
Shouldn't this be https://github.com/OWASP/Amass/blob/master/doc/amass.gif?

Reply:
Yep!